orc/src/main/java/org/apache/iceberg/orc/OrcValueWriter.java

```diff
@@ -19,20 +19,28 @@
 
 package org.apache.iceberg.orc;
 
-import java.io.IOException;
-import org.apache.orc.storage.ql.exec.vector.VectorizedRowBatch;
+import org.apache.orc.storage.ql.exec.vector.ColumnVector;
 
-/**
- * Write data value of a schema.
- */
 public interface OrcValueWriter<T> {
 
+  Class<T> getJavaClass();
```

Comment:
What will this be used for? I don't see anything calling it in this commit.

Reply:
It's used for reading the field from `Record` and casting the value to target class , see here: https://github.com/apache/iceberg/pull/1197/files#diff-69c0f1e45966d2eb49a315fe32734cf5R125.